### PR TITLE
docs: Support more tags for posts

### LINKED ISSUE #744: support more tags for posts
**Actual Behaviour**

When I visit this post: https://apisix.apache.org/blog/2021/11/12/apisix-datadog , I noticed there has only one Tag called `Technology`. It's too common to use this as the only one tag IMO, and I just checked this page's Meta data, see

image

**Expected Behaviour**

After comparing with other sites' blogs, I would recommend we fulfill more tags. For the post above, I would recommend using `DataDog,Monitoring,Observability` as the tags.

Not only this post, we should update all posts.

### PR DESCRIPTION
**Pull request by adarrssh**
Fixes: #744 

Changes: Added more tags to blog posts.

### CODE REVIEW

**Comment by juzhiyuan** [comment]
cc @yzeng25 to have a review 😄

File: website/blog/2021/11/12/apisix-datadog.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ keywords:
 - Observability
 - Cloud Monitoring
 description: Apache APISIX recently released a new plugin APISIX-Datadog, to provide a deeper integration with Datadog. This article introduces the APISIX-Datadog Plugin and its capabilities.
-tags: [Technology]
+tags: [Technology,DataDog,Monitoring]
```
Comment (yzeng25) [inline website/blog/2021/11/12/apisix-datadog.md]: I would suggest remove `Datadog` as discussed in #744
```suggestion
tags: [Technology,Monitoring]
```

File: website/blog/2021/11/30/use-apisix-ingress-in-kubesphere.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ keywords:
 - Ingress Controller
 - Monitor
 description: This article will take Apache APISIX Ingress Controller as an example to show you in detail how to quickly use different types of gateways and status monitoring for Kubernetes clusters through KubeSphere.
-tags: [Technology]
+tags: [Technology,Monitoring]
```
Comment (yzeng25) [inline website/blog/2021/11/30/use-apisix-ingress-in-kubesphere.md]: Might add Kubernetes as well :)
```suggestion
tags: [Technology,Monitoring,Kubernetes]
```

File: website/blog/2021/12/08/apisix-integrate-rocketmq-logger-plugin.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ keywords:
 - Log processing
 - Monitoring function
 description: This article will introduce the latest integration of Apache APISIX and Apache RocketMQ rocketmq-logger plug-in features and use. With this plug-in, you can connect to the RocketMQ cluster more easily when using Apache APISIX.
-tags: [Technology]
+tags: [Technology,Monitoring]
```
Comment (yzeng25) [inline website/blog/2021/12/08/apisix-integrate-rocketmq-logger-plugin.md]:
```suggestion
tags: [Technology,Message Queue]
```

File: website/blog/2021/12/13/monitor-apisix-ingress-controller-with-prometheus.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ keywords:
 - Metrics
 - Grafana
 description: This article introduces the relevant steps of how to use Prometheus to monitor APISIX Ingress Controller and the display effect of some indicators.
-tags: [Technology]
+tags: [Technology,Prometheus]
```
Comment (yzeng25) [inline website/blog/2021/12/13/monitor-apisix-ingress-controller-with-prometheus.md]:
```suggestion
tags: [Technology,Monitoring]
```

File: website/blog/2021/12/24/open-policy-agent.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ keywords:
 - OPA
 - API Gateway
 description: This article introduces the opa plug-in as an example of HTTP API and details how to integrate Apache APISIX with OPA to decouple the authentication authorization of back-end services.
-tags: [Technology]
+tags: [Technology,Ecosystem]
```
Comment (yzeng25) [inline website/blog/2021/12/24/open-policy-agent.md]:
```suggestion
tags: [Technology,Authentication]
```

**Comment by yzeng25** [comment]
Hi @adarrssh , thank you for your pull request! 

Do you mind adding tags to blogs in `website/i18n/zh/docusaurus-plugin-content-blog` as well?

**Comment by adarrssh** [comment]
Sure @yzeng25 😊.  I will update the blogs of  website/i18n/zh/docusaurus-plugin-content-blog.

**Comment by yzeng25** [comment]
> Sure @yzeng25 😊. I will update the blogs of website/i18n/zh/docusaurus-plugin-content-blog.

Awesome! The only left-over thing is to rebase your commit messages to `docs: Support more tags for posts`, and change the PR title to `docs: Support more tags for posts` to satisfy the semantic check.

**Review by yzeng25 — APPROVED** [review]

**Comment by yzeng25** [comment]
> cc @yzeng25 to have a review 😄

Hi @juzhiyuan , can we get 1 approve to start the remaining workflows?

**Review by juzhiyuan — APPROVED** [review]

**Comment by adarrssh** [comment]
Hey @yzeng25, can we get 1 approval to merge this pull request 😊.

**Review by guoqqqi — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Comment by lijing-21** [comment]
Hi @adarrssh , thank you for your contribution!

Here is the Contributor T-shirt form[1], if you're interested, kindly take a look :)

[1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt

**Comment by adarrssh** [comment]
Hey @lijing-21, Thank you for informing me. I have filled out the T-shirt form 😊.

**Pull request merged**